src/vnsw/opencontrail-vrouter-netns/opencontrail_vrouter_netns/vrouter_hyperv.py

```diff
@@ -0,0 +1,469 @@
+        create_parser.add_argument(
+            "--vm_location",
+            required=True,
+            help="Location of gateway VM")
+        create_parser.add_argument(
+            "--vhd_path",
+            required=True,
+            help="Path of VHD of VM")
+        create_parser.add_argument(
+            "--mgmt_vswitch_name",
+            required=True,
+            help="Name of management virtual switch")
+        create_parser.add_argument(
+            "--vrouter_vswitch_name",
+            required=True,
+            help="Name of vRouter virtual switch")
+        create_parser.add_argument(
+            "vm_id",
+            help="Virtual machine UUID")
+        create_parser.add_argument(
+            "vmi_left_id",
+            help="Left virtual machine interface UUID")
+        create_parser.add_argument(
+            "vmi_right_id",
+            help="Right virtual machine interface UUID")
+        create_parser.add_argument(
+            "--vmi-left-mac",
+            default=None,
+            help=("Left virtual machine interface MAC. Default: automatically "
+                  "generated by the system"))
+        create_parser.add_argument(
+            "--vmi-left-ip",
+            default=None,
+            help=("Left virtual machine interface IPv4 and mask "
+                  "(ie: a.a.a.a/bb). Default mask to /32"))
+        create_parser.add_argument(
+            "--vmi-right-mac",
+            default=None,
+            help=("Right virtual machine interface MAC. Default: "
+                  "automatically generated by the system"))
+        create_parser.add_argument(
+            "--vmi-right-ip",
+            default=None,
+            help=("Right virtual machine interface IPv4 and mask "
+                  "(ie: a.a.a.a/bb). Default mask to /32"))
+        create_parser.add_argument(
+            "--gw-ip",
+            default=None,
+            help=("Gateway IP for Virtual Network"))
+        create_parser.set_defaults(func=self.create)
+
+        destroy_parser = subparsers.add_parser('destroy')
+        destroy_parser.add_argument(
+            "vm_id",
+            help="Virtual machine UUID")
+        destroy_parser.add_argument(
+            "vmi_left_id",
+            help="Left virtual machine interface UUID")
+        destroy_parser.add_argument(
+            "vmi_right_id",
+            help="Right virtual machine interface UUID")
+        destroy_parser.set_defaults(func=self.destroy)
+
+        self.args = parser.parse_args(remaining_argv)
+
+    def create(self):
+        vm_id = validate_uuid(self.args.vm_id)
+
+        nic_left = {}
+        if uuid.UUID(self.args.vmi_left_id):
+            nic_left['uuid'] = validate_uuid(self.args.vmi_left_id)
+            if self.args.vmi_left_mac:
+                nic_left['mac'] = netaddr.EUI(self.args.vmi_left_mac,
+                                              dialect=netaddr.mac_eui48) # does eui48 break sth?
```

Comment:
It would be nice if both `AA-BB` and `aa:bb` were supported. One is used everywhere in Windows and another is used everywhere in Contrail.